wp-content/themes/access/src/js/modules/screener.js

```diff
@@ -75,6 +78,33 @@ class Screener {
       e.preventDefault();
       const hash = window.location.hash;
       const $section = $(hash);
+      let title = $section.find('[data-js="question"]');
+      let prevStep;
+
+      this._history.push(hash);
+
+      if (this._history.length > 1) {
+         prevStep = this._history[this._history.length - 2];
+      }
+
+      let stateObj = {
+        prevStep: prevStep,
+        step: `${hash}`,
+        persons: this._household._attrs.members,
+        title: title[0].innerText,
+        person: []
+      };
+
+      this._people.forEach(function(person, index) {
+        stateObj.person.push({
+          index: index,
+          headOfHousehold: person._attrs.headOfHousehold,
+          income: (!person._attrs.incomes.length) ? false : true,
+          expenses: (!person._attrs.expenses.length) ? false : true
+        });
+      });
+
+      window.history.pushState(stateObj, '', '');
```

Comment:
remove empty strings? `window.history.pushState(stateObj);`

Reply:
Although these are optional parameters, it would throw an error if empty parameters are not passed into the function.```TypeError: Not enough arguments to History.pushState.```

Reply:
Awesome thanks for clarifying